pkg/node/node.go

```diff
@@ -438,6 +451,58 @@ func (n *Node) publishSignature(
 	}
 }
 
+func (n *Node) waitSignerPublicationDelay(
```

Comment:
Can we add simple docs to this function? Mostly what's already said in docs for `signerPublicationDelayStep`.